src/navigations/routes.tsx

```diff
@@ -104,3 +103,5 @@ export const allRoutes = [
   ...routes,
   ...Object.values(additionalRoutesWithoutNavMenuNavigation),
 ];
+
+// export const allRoutes = routes;
```

Comment:
Why is this commented out?

Reply:
Because it should be removed! (I missed it, fixing...)

Reply:
(Removed!)